src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkQuantumStorage.java

```diff
@@ -108,10 +119,12 @@ public class NetworkQuantumStorage extends SlimefunItem implements DistinctiveIt
 
     private final List<Integer> slotsToDrop = new ArrayList<>();
     private final int maxAmount;
+    private final boolean supportsCustomMaxAmount;
 
-    public NetworkQuantumStorage(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, int maxAmount) {
```

Comment:
Just a suggestion, you could keep this constructor and use a this call to make false the default, that way you aren't putting false on all old constructors

Reply:
TypeScript is normally my chosen language, so I wanted to `= false` default it, but that'd need an override constructor afaik in Java. Could have a set for true and default to false, but it might get "forgotten" instead of being explicitly yes or no - but it depends on which standards are followed in this project